doc/releases/reef.rst

```diff
@@ -0,0 +1,266 @@
+====
+Reef
+====
+
+Reef is the 18th stable release of Ceph. It is named after the reef squid
+(Sepioteuthis).
+
+v18.2.0 Reef
+============
+
+This is the first stable release of Ceph Reef.
+
+CephFS
+~~~~~~
+
+* CephFS: The ``mds_max_retries_on_remount_failure`` option has been renamed to
+  ``client_max_retries_on_remount_failure`` and moved from ``mds.yaml.in`` to
+  ``mds-client.yaml.in``. This change was made because the option has always
+  been used only by the MDS client.
+* CephFS: It is now possible to delete the recovered files in the
+  ``lost+found`` directory after a CephFS post has been recovered in accordance
+  with disaster recovery procedures.
+* The ``AT_NO_ATTR_SYNC`` macro has been deprecated in favor of the standard
+  ``AT_STATX_DONT_SYNC`` macro. The ``AT_NO_ATTR_SYNC`` macro will be removed
+  in the future.
+
+Dashboard
+~~~~~~~~~
+
+* There is a new Dashboard page with improved layout. Active alerts
+  and some important charts are now displayed inside cards.
+
+* Cephx Auth Management: There is a new section dedicated to listing and
+  managing Ceph cluster users.
+  
+* RGW Server Side Encryption: The SSE-S3 and KMS encryption of RGW buckets can
+  now be configured at the time of bucket creation.
+
+* RBD Snapshot mirroring: Snapshot mirroring can now be configured through UI.
+  Snapshots can now be scheduled.
+  
+* 1-Click OSD Creation Wizard: OSD creation has been broken into three options:
+
+  #. Cost/Capacity Optimized: Use all HDDs
+
+  #. Throughput Optmized: HDDs and SSDs combined
+
+  #. IOPS Optimized: Use all NVME SSDs
+
+  The current OSD-creation form has been moved to the Advanced section.
+
+* Centralized Logging: There is now a view that collects all the logs from
+  the Ceph cluster.
+
+* Accessibility WCAG-AA: Dashboard is WCAG 2.1 level A compliant and therefore
+  improved for blind and visually impaired Ceph users.
+
+* Monitoring & Alerting
+
+      * Ceph-exporter: Now the metrics are exported with ceph-exporter, which
```

Comment:
For ceph-exporter it should be explicitly mentioned that it only exports the performance counters.
`Now the performance metrics for ceph daemons are exported by ceph-exporter, which ....`

Reply:
I've made this change.

Reply:
LGTM, thanks!